packages/react-components/src/templates/WorkingGroupHeader.tsx

```diff
@@ -99,66 +138,113 @@ const WorkingGroupPageHeader: React.FC<WorkingGroupPageHeaderProps> = ({
-        }#${membersListElementId}`}
-      />
-      {pointOfContact && (
-        <div css={pointOfContactStyles}>
-          <Link
-            buttonStyle
-            small
-            primary
-            href={`${createMailTo(pointOfContact.user.email)}`}
-          >
-            Contact PM
-          </Link>
-        </div>
-      )}
-    </section>
-    <div css={rowStyles}>
-      {externalLink && (
-        <ExternalLink
-          full
-          label="Working Group Folder"
-          href={externalLink}
-          size="large"
+}) => {
+  const currentUserCRN = useCurrentUserCRN();
+  const isWorkingGroupProjectManager = isProjectManager(
+    currentUserCRN,
+    leaders,
+  );
+
+  return (
+    <header css={containerStyles}>
+      <div css={titleStyle}>
+        <Display styleAsHeading={2}>{title}</Display>
+        {complete && (
+          <StateTag accent="green" icon={successIcon} label="Complete" />
+        )}
+      </div>
+      <section css={contactSectionStyles}>
+        <UserAvatarList
+          members={[...leaders, ...members].map((member) => member.user)}
+          fullListRoute={`${
+            network({})
+              .workingGroups({})
+              .workingGroup({ workingGroupId: id })
+              .about({}).$
+          }#${membersListElementId}`}
         />
-      )}
-      <div css={lastUpdatedStyles}>
-        <Caption asParagraph accent="lead">
-          Last updated: {formatDistance(new Date(), new Date(lastModifiedDate))}{' '}
-          ago
-        </Caption>
+        {pointOfContact && !isWorkingGroupProjectManager && (
+          <div css={pointOfContactStyles}>
+            <Link
+              buttonStyle
+              small
+              primary
+              href={`${createMailTo(pointOfContact.user.email)}`}
+            >
+              Contact PM
+            </Link>
+          </div>
+        )}
+
+        {isEnabled('WORKING_GROUP_SHARED_OUTPUT_BTN') &&
+          isWorkingGroupProjectManager && (
+            <div css={createStyles}>
+              <DropdownButton
+                buttonChildren={() => (
+                  <span css={dropdownButtonStyling}>
+                    {plusIcon}
+                    Share an output
+                  </span>
+                )}
+              >
+                {{
+                  item: <>{article} Article</>,
+                  href: '#',
+                }}
+                {{
+                  item: <>{bioinformatics} Bioinformatics</>,
+                  href: '#',
+                }}
+                {{
+                  item: <>{dataset} Dataset</>,
+                  href: '#',
+                }}
+                {{
+                  item: <>{labResource} Lab Resource</>,
+                  href: '#',
+                }}
+                {{
+                  item: <>{protocol} Protocol</>,
+                  href: '#',
+                }}
+                {{
+                  item: <>{reportIcon} CRN Report</>,
```

Comment:
I believe this is not the expected icon

<img width="974" alt="Screenshot 2023-01-02 at 07 29 03" src="https://user-images.githubusercontent.com/16595804/210219900-b152c79d-272f-41ae-abdf-2f50e16472e8.png">